compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/TypeChecker.java

```diff
@@ -7049,7 +7049,7 @@ private void checkFunctionInvocationExpr(BLangInvocation iExpr, AnalyzerData dat
         iExpr.symbol = funcSymbol;
         checkInvocationParamAndReturnType(iExpr, data);
 
-        if (langLibPackageID && !iExpr.argExprs.isEmpty()) {
+        if (langLibPackageID && !iExpr.argExprs.isEmpty() && iExpr.argExprs.get(0).getBType() != null) {
```

Comment:
This happens because the first arg was not visited. I think we have to check the reason for that and fixing at that point.

Reply:
They are not visited since these are unexpected arguments. We can visit them before creating the errors in `checkInvocationArgs`. If we're doing `checkTypeParamExpr` there, should we have `neverType` as expected type?

Reply:
We have to try and see. We can set the `noType` as the expected type which means call checkExpr without the expected type.